fuel/src/main/kotlin/fuel/util/Delegates.kt

```diff
@@ -1,18 +1,19 @@
 package fuel.util
 
 import kotlin.properties.ReadWriteProperty
+import kotlin.reflect.KProperty
```

Comment:
I m kind of a bit surprise that we can do this without `kotlin.reflect` dependency.